lib/Conversion/ExportVerilog/ExportVerilog.cpp

```diff
@@ -308,9 +308,9 @@ static bool isZeroBitType(Type type) {
   if (auto inout = type.dyn_cast<hw::InOutType>())
     return isZeroBitType(inout.getElementType());
   if (auto uarray = type.dyn_cast<hw::UnpackedArrayType>())
-    return isZeroBitType(uarray.getElementType());
+    return uarray.getSize() == 0 || isZeroBitType(uarray.getElementType());
   if (auto array = type.dyn_cast<hw::ArrayType>())
-    return isZeroBitType(array.getElementType());
+    return array.getSize() == 0 || isZeroBitType(array.getElementType());
```

Comment:
These lines are unrelated to ExportVerilog simplification but necessary to fix integration failures. It seems zero element arrays were not considered as zero bit.